config.yaml

```diff
@@ -75,6 +72,8 @@ clouds:
     # Accounts to share newly created AMIs with
     test_accounts:
       - "013116697141"
+    # Temporary hack for AWS aarch64; see comment in `build-arch.Jenkinsfile`.
+    aarch64_serial_console_hack: true
```

Comment:
I *think* I'd like to put this somewhere different. I know this is temporary so it doens't really matter and I probably shouldn't argue this point, but...

This section to me is more about the cloud upload (i.e. not affecting the generated artifact (file), but rather the image creation in the cloud).

I'd be fine with a toplevel `aws_aarch64_serial_console_hack: true`, especially since it's temporary. This code/patch is completely a hack right now and hopefully we don't have to do stuff like this in the future.

Reply:
I'm fine with having it be at the top level.

Reply:
I think this still needs to be moved in the config to the top level.

Reply:
Ahh missed the `git add`.